ocis/pkg/command/storageappprovider.go

```diff
@@ -0,0 +1,45 @@
+// +build !simple
+
+package command
+
+import (
+	"github.com/micro/cli/v2"
+	"github.com/owncloud/ocis/ocis-pkg/config"
+	"github.com/owncloud/ocis/ocis/pkg/register"
+	"github.com/owncloud/ocis/storage/pkg/command"
+	svcconfig "github.com/owncloud/ocis/storage/pkg/config"
+	"github.com/owncloud/ocis/storage/pkg/flagset"
+)
+
+// StorageAppProviderCommand is the entrypoint for the reva-app-provider command.
+func StorageAppProviderCommand(cfg *config.Config) *cli.Command {
```

Comment:
Why do we have both storageappprovider and appprovider?

Reply:
I removed the "appprovider" one since we have prefixed all service coming from reva with "storage". Don't know if that should be continued forever but I did it for now